flygym/simulation.py

```diff
@@ -218,10 +218,33 @@ def step(
         obs, reward, terminated, truncated, info = {}, {}, {}, {}, {}
 
         for fly in self.flies:
-            k = fly.name
-            obs[k], reward[k], terminated[k], truncated[k], info[k] = fly.post_step(
-                self
-            )
+            key = fly.name
+
+            obs[key], _, _, _, _ = fly.post_step(self)
+            reward[key] = fly.get_reward()
```

Comment:
@tkclam Is there a particular reason why you overwrote `reward[key]`, `terminated[key]`, `truncated[key]`, and `info[key]` with `fly.get_reward()` etc, instead of simply using the output from `fly.post_step(self)`?

Reply:
Or is it a result of some automerge somewhere? Can I changed it to the following, as is the case in your commit 260a5e3?

```
obs[k], reward[k], terminated[k], truncated[k], info[k] = fly.post_step(self)
```